it/uk/gov/hmrc/traderservices/controllers/TraderServicesFrontendISpec.scala

```diff
@@ -1492,83 +1492,84 @@ class TraderServicesFrontendISpec
+                  "396f101dd52e8b2ace0dcf5ed09b1d1f030e608938510ce46e7a5c7a4e775100",
+                  "test.pdf",
+                  "application/pdf"
+                )
               )
             ),
-            "TBC"
+            acknowledged = true
           )
-        }
+        )
+        givenAuthorisedForEnrolment(Enrolment("HMRC-XYZ", "EORINumber", "foo"))
+        givenCreateCaseApiRequestSucceeds()
+
+        val result = await(request("/pre-clearance/create-case").post(""))
+
+        result.status shouldBe 200
+        result.body should include(htmlEscapedMessage("view.create-case-confirmation.title"))
+        result.body should include(htmlEscapedMessage("view.create-case-confirmation.heading"))
+        journey.getState shouldBe CreateCaseConfirmation(
+          TestData.exportDeclarationDetails,
+          TestData.fullExportQuestions(dateTimeOfArrival),
+          Seq(
+            UploadedFile(
+              "https://bucketName.s3.eu-west-2.amazonaws.com?1235676",
+              ZonedDateTime.parse("2018-04-24T09:30:00Z"),
+              "396f101dd52e8b2ace0dcf5ed09b1d1f030e608938510ce46e7a5c7a4e775100",
+              "test.pdf",
+              "application/pdf"
+            )
+          ),
+          "A1234567890"
+        )
       }
+    }
 
-      "GET /pre-clearance/confirmation" should {
-        "show the confirmation page" in {
-          implicit val journeyId: JourneyId = JourneyId()
-          val dateTimeOfArrival = dateTime.plusDays(1).truncatedTo(ChronoUnit.MINUTES)
-          val state = CreateCaseConfirmation(
-            TestData.exportDeclarationDetails,
-            TestData.fullExportQuestions(dateTimeOfArrival),
-            Seq(
-              UploadedFile(
-                "https://bucketName.s3.eu-west-2.amazonaws.com?1235676",
-                ZonedDateTime.parse("2018-04-24T09:30:00Z"),
-                "396f101dd52e8b2ace0dcf5ed09b1d1f030e608938510ce46e7a5c7a4e775100",
-                "test.pdf",
-                "application/pdf"
-              )
-            ),
-            "TBC"
-          )
-          journey.setState(state)
-          givenAuthorisedForEnrolment(Enrolment("HMRC-XYZ", "EORINumber", "foo"))
+    "GET /pre-clearance/confirmation" should {
+      "show the confirmation page" in {
+        implicit val journeyId: JourneyId = JourneyId()
+        val dateTimeOfArrival = dateTime.plusDays(1).truncatedTo(ChronoUnit.MINUTES)
+        val state = CreateCaseConfirmation(
+          TestData.exportDeclarationDetails,
+          TestData.fullExportQuestions(dateTimeOfArrival),
+          Seq(
+            UploadedFile(
+              "https://bucketName.s3.eu-west-2.amazonaws.com?1235676",
+              ZonedDateTime.parse("2018-04-24T09:30:00Z"),
+              "396f101dd52e8b2ace0dcf5ed09b1d1f030e608938510ce46e7a5c7a4e775100",
+              "test.pdf",
+              "application/pdf"
+            )
+          ),
+          "TBC"
```

Comment:
Can this become A1234567890 as well?

Reply:
this is whatever is in the state, the state doesn't change when we call show...